exercises/palindrome-products/palindrome_products_test.go

```diff
@@ -37,10 +38,14 @@ var testData = []struct {
 		""},
 	{4, 10, Product{}, Product{}, "No palindromes"},
 	{10, 4, Product{}, Product{}, "fmin > fmax"},
-	/* bonus curiosities.  (can a negative number be a palindrome?
-	// most say no
-	{-99, -10, Product{}, Product{}, "Negative limits"},
-	// but you can still get non-negative products from negative factors.
+
+	/* Bonus curiosities.  Can a negative number be a palindrome? Most say no
+	{-99, -10,
+		Product{},
+		Product{},
+	"Negative limits"},
```

Comment:
it definitely makes sense. 
I have now put the bonus data in a different slice (called bonusData) and I put this commented line (`//testData = append(testData, bonusData...)`) inside the test function `TestPalindromeProducts`, in the first line.

The bonus slice remains not commented. And doing this I found a compilation error with the bonus data (related to the last element of the bonus slice):

> cannot use ^uint(0) >> 1 (type uint) as type int in field value

The problem now is that if the bonus slice remains uncommented, this line won't make the test compile.